cypress/integration/dashboard/events.js

```diff
@@ -83,10 +81,6 @@ describe('events dashboard', () => {
     cy.findByRole('textbox', { name: 'Tags (separated by a comma)' }).type(
       'Test, Event, Tag',
     );
-
-    cy.findByLabelText(/^Start at/).type(testEvent.startAt);
-
-    cy.findByLabelText(/^End at/).type(testEvent.endAt);
```

Comment:
```
    cy.findByLabelText(/^Start at/)
      .clear()
      .type(testEvent.startAt)
      .type('{esc}');
    cy.findByLabelText(/^End at/)
      .clear()
      .type(testEvent.endAt)
      .type('{esc}');
```

This should work.  The escaping being necessary to hide the picker and let Cypress get to the next field.